Upgrade Community Calls/Call_001.md

```diff
@@ -0,0 +1,271 @@
+# Shanghai/Capella Community Call #1 
+### Jan 20, 2023, 15:00-16:00 UTC
+### Meeting Duration: 1 hour 
+### [GitHub Agenda](https://github.com/ethereum/pm/issues/708)
+### [Video of the meeting](https://www.youtube.com/live/uTWpSYn4MA8?feature=share)
+### Moderator: Tim Bieko
+### Notes: Alen (Santhosh)
+
+
+**Tim**
+* Good morning, afternoon, everyone, depending on where you are. first Chappelle community call. so I guess, these are, are, are helpful to just like, answer people's questions about the upgrade. we started doing them with the merge and, given the introduction of withdrawals, yeah, it makes sense to, do them as well so that validators and, and just node operators and infrastructure providers could come and, and ask questions. didn't have too much on the agenda today. I think,  if we have Danny and myself here, we could maybe both give a quick overview of like the entire upgrade so you get withdrawals, but there's a couple more things as well. 
```

Comment:
```suggestion
* Good morning, afternoon, everyone, depending on where you are. first Shappelle community call. so I guess, these are helpful to just like, answer people's questions about the upgrade. we started doing them with the merge and, given the introduction of withdrawals, yeah, it makes sense to, do them as well so that validators and, and just node operators and infrastructure providers could come and, and ask questions. didn't have too much on the agenda today. I think,  if we have Danny and myself here, we could maybe both give a quick overview of like the entire upgrade so you get withdrawals, but there's a couple more things as well. 
```